scripts/coverage_helper.py

```
@@ -0,0 +1,82 @@
+"""Utility helpers for the coverage pipeline used by the pdm test scripts."""
+# SPDX-FileCopyrightText: © Jens Bergmann and contributors
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# SPDX-ArtifactOfProjectHomePage: https://github.com/Second-Hand-Friends/kleinanzeigen-bot/
+from __future__ import annotations
+
+import argparse
+import os
+import subprocess  # noqa: S404 subprocess usage is limited to known internal binaries
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `S404`)

Remove unused `noqa` directive

(RUF100)